core/src/main/java/com/rafaskoberg/gdx/parrot/ParrotSettings.java

```diff
@@ -16,14 +17,24 @@ public class ParrotSettings {
     public int loudnessExponentialCurve = 3;
 
     /**
-     * In the spatial system, distances beyond this limit will have the maximum amount of reduction. Defaults to 15
+     * In the spatial system, distances beyond this limit will be based off {@link float distanceRolloffMinGain}. Defaults to 15

```

Comment:
These member links didn't show up on IntelliJ for me, and I think it's because of the `float` thing instead of referring to a local member with `#`, hopefully that works.

```suggestion
     * In the spatial system, distances beyond this limit will be based off {@link #distanceRolloffMinGain}. Defaults to 15

```